msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ui/MessageForumStatisticsBean.java

```diff
@@ -2184,6 +2184,9 @@ private String getExternalParameterByKey(String parameterId)
 		return (String) paramMap.get(parameterId);    
 	}
 
+	public boolean getIsAuthor() {
+		return selectedSiteUserId == userDirectoryService.getCurrentUser().getId();
```

Comment:
Is this comparing strings?!?! Is this okay?

Reply:
I think it is, I kinda don't like it but I think it works fine.

Reply:
"comparing the reference of the given strings, depending on if they are referring to the same objects." ... how can they refer to the same object?

Reply:
Looks like it compares the String value. Is this causing an issue in any instance?

Reply:
Sorry this was fixed in SAK-45854 ef3817fd098e396ae722425cb3884f4da737667d